### PR TITLE
[Android] Fix ProgressRing fails to display

### LINKED ISSUE #2762: [Android] ProgressRing is not visible
## Current behavior

The `ProgressRing` control is not visible.

May be related to https://github.com/unoplatform/uno/issues/2761

## How to reproduce it (as minimally and precisely as possible)

TBD
https://dev.azure.com/nventive/Umbrella/_workitems/edit/176246

## Environment

Nuget Package:

Package Version(s):

Affected platform(s):

- [ ] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [ ] macOS
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

Visual Studio:

- [ ] 2017 (version: )
- [ ] 2019 (version: )
- [ ] for Mac (version: )

Relevant plugins:

- [ ] Resharper (version: )

### PR DESCRIPTION
**Pull request by davidjohnoliver**
ProgressRing inside a StackPanel was failing to display because its desired size wasn't set. Fix this by introducing a flag to distinguish between managed views that have their own Layouter, and those that don't (like ProgressRing).

GitHub Issue (If applicable): fixes #2762 

## PR Type

What kind of change does this PR introduce?

- Bugfix

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](https://github.com/unoplatform/uno/blob/master/README.md#uno-features)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [x] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Updated the [Release Notes](https://github.com/unoplatform/uno/tree/master/doc/ReleaseNotes)
- [x] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UI/Mixins/Android/FrameworkElementMixins.tt [hunk 1]
```diff
@@ -3,14 +3,14 @@
...
+	AddClass("Windows.UI.Xaml", "FrameworkElement", overridesAttachedToWindow:true, overridesOnLayout: false, isUnoMotionTarget: true, hasLayouter: true);
 	AddClass("Uno.UI.Controls.Legacy", "ListView");
-	AddClass("Windows.UI.Xaml.Controls", "Image", hasNewMaxWidthHeight: true, hasAutomationPeer: true);
+	AddClass("Windows.UI.Xaml.Controls", "Image", hasNewMaxWidthHeight: true, hasAutomationPeer: true, hasLayouter: true);
 	AddClass("Uno.UI.Controls.Legacy", "HorizontalGridView");
 	AddClass("Uno.UI.Controls.Legacy", "HorizontalListView");
 	AddClass("Windows.UI.Xaml.Controls", "ProgressRing");
-	AddClass("Windows.UI.Xaml.Controls", "ScrollContentPresenter", callsBaseOnLayout: false);
-	AddClass("Windows.UI.Xaml.Controls", "NativePagedView");
+	AddClass("Windows.UI.Xaml.Controls", "ScrollContentPresenter", callsBaseOnLayout: false, hasLayouter: true);
```
Comment (dr1rrb) [inline src/Uno.UI/Mixins/Android/FrameworkElementMixins.tt]: Not sure if it's expected, the `ScrollContentPresenter` and `NativePagedView` do not have the flag `hasLayouter` on iOS and WASM

Comment (davidjohnoliver) [reply src/Uno.UI/Mixins/Android/FrameworkElementMixins.tt]: It's because they has not

**Comment by jeromelaban** [comment]
@Mergifyio backport   release/beta/2.1

**Comment by jeromelaban** [comment]
@Mergifyio backport release/beta/2.1

**Pull request merged**